graphql/validate.lua

```diff
@@ -270,6 +431,15 @@ local visitors = {
     rules = { rules.uniqueInputObjectFields }
   },
 
+  -- <inputObject>
+  --
+  -- {
+  --   kind = 'inputObject',
+  --   values = list of {
+  --     name = <string>,
+  --     value = <value>,
+  --   },
+  -- }
```

Comment:
An input object may contain a another input object as a field. It is not reflected there.

Reply:
From above:

```lua
  -- <value> is one of the following:
  --  
  -- * <variable>
  -- * <inputObject>
  -- * <list>
  -- * <enum>
  -- * <string>
  -- * <boolean>
  -- * <float>
  -- * <int>
  -- * <null>
```

So an inputObject actually can contain an inputObject as a field according to the AST description. Everyting looks okay here.